web/src/locales/fr/translation.json

```
@@ -0,0 +1,242 @@
+    "subtitle": "Moniteur d'état des services"
+  },
+  "auth": {
+    "logout": "Se déconnecter"
+  },
+  "languages": {
+    "en": "Anglais",
+    "fr": "Français"
+  },
+  "status": {
+    "online": "En ligne",
+    "offline": "Hors ligne",
```

Biome (2.5.6)

[error] The key offline was already declared.

(lint/suspicious/noDuplicateObjectKeys)